engine/schema/src/main/java/com/cloud/upgrade/dao/Upgrade41500to41510.java

```diff
@@ -250,130 +249,6 @@ private void correctGuestOsNames(final Connection conn) {
         guestOsMapper.updateGuestOsNameFromMapping("OpenSUSE Linux (32 bit)", mapping);
     }
```

Comment:
@weizhouapache , your comment in https://github.com/apache/cloudstack/pull/7095#issuecomment-1382247322 means that also this method must be moved? It contains a call to `updateGuestOsNameFromMapping()` as well.

Reply:
double-checked just now.

both `addGuestOsAndHypervisorMappings` and `updateGuestOsIdInHypervisorMapping` call the method `getGuestOsId`, they need to be moved.

`updateGuestOsNameFromMapping` is correct. sorry for my mistake